test/data/manifests/fedora_34-x86_64-vmdk-boot.json

```diff
@@ -9208,17 +11839,17 @@
   "image-info": {
     "/etc/resolv.conf": [],
     "boot-environment": {
-      "kernelopts": "root=UUID=76a22bf4-f153-4541-b6c7-0332c0dfaeac"
+      "saved_entry": "ffffffffffffffffffffffffffffffff-5.15.13-100.fc34.x86_64"
     },
     "bootloader": "grub",
     "bootmenu": [
       {
         "grub_arg": "--unrestricted",
         "grub_class": "fedora",
         "grub_users": "$grub_users",
-        "initrd": "/boot/initramfs-5.15.13-100.fc34.x86_64.img",
-        "linux": "/boot/vmlinuz-5.15.13-100.fc34.x86_64",
-        "options": "root=UUID=76a22bf4-f153-4541-b6c7-0332c0dfaeac ro no_timer_check net.ifnames=0 console=tty1 console=ttyS0,115200n8",
```

Comment:
kernel options change....

Reply:
Fixed. The package is back,Updated. Please review.